tikv-client/src/main/java/com/pingcap/tikv/TiConfiguration.java

```diff
@@ -167,278 +151,10 @@ public int getScanBatchSize() {
     return DEF_SCAN_BATCH_SIZE;
   }
 
-  boolean isIgnoreTruncate() {
-    return ignoreTruncate;
-  }
-
-  public TiConfiguration setIgnoreTruncate(boolean ignoreTruncate) {
-    this.ignoreTruncate = ignoreTruncate;
-    return this;
-  }
-
-  boolean isTruncateAsWarning() {
-    return truncateAsWarning;
-  }
-
-  public TiConfiguration setTruncateAsWarning(boolean truncateAsWarning) {
-    this.truncateAsWarning = truncateAsWarning;
-    return this;
-  }
-
-  public int getMaxFrameSize() {
-    return maxFrameSize;
-  }
-
-  public TiConfiguration setMaxFrameSize(int maxFrameSize) {
-    this.maxFrameSize = maxFrameSize;
-    return this;
```

Comment:
does lombok will return TiConfiguration in setMaxFrameSize? if not, we need to see if this change will affect other places.
plz check other code with the same problems in this pr

Reply:
https://projectlombok.org/features/experimental/Accessors
@Accessor(chain = true) will do this

Reply:
@Accessors(chain = true) can change setter method return value ,more info you can see [lombok/Accessors](https://projectlombok.org/features/experimental/Accessors)